execution_engine_testing/tests/src/test/regression/ee_1217_mint_proxy.rs

```diff
@@ -88,47 +141,17 @@ fn faucet_should_create_account() {
     )
     .build();
 
-    builder.exec(faucet_request).commit().expect_success();
-
-    let alice_account = {
-        let tmp = builder.query(None, Key::Account(*ALICE_ADDR), &[]).unwrap();
-        tmp.as_account().cloned().unwrap()
-    };
-
-    let balance = builder.get_purse_balance(alice_account.main_purse());
-
-    assert_eq!(balance, faucet_request_amount);
-
-    let faucet_request = ExecuteRequestBuilder::contract_call_by_hash(
-        *DEFAULT_ACCOUNT_ADDR,
-        faucet_hash.into(),
-        CONTRACT_FAUCET_ENTRYPOINT,
-        runtime_args! {
-            ARG_TARGET => *ALICE_ADDR,
-            ARG_AMOUNT => faucet_request_amount,
-        },
-    )
-    .build();
-
     builder.exec(faucet_request).commit();
 
-    let error = {
-        let response = builder
-            .get_exec_results()
-            .last()
-            .expect("should have last exec result");
-        let exec_response = response.last().expect("should have response");
-        exec_response.as_error().expect("should have error")
-    };
-    assert_matches!(
-        error,
-        engine_state::Error::Exec(execution::Error::Revert(ApiError::User(1)))
-    );
+    match builder.get_error() {
+        Some(engine_state::Error::Exec(execution::Error::Revert(ApiError::Mint(20)))) => {}
```

Comment:
If possible; for better readability

```suggestion
        Some(engine_state::Error::Exec(execution::Error::Revert(api_error))) if api_error == ApiError::from(mint::Error::InvalidContext) => {}
```

Reply:
:+1:

Reply:
Addressed